src/inject/GPMInject/interface/mini.js

```diff
@@ -7,12 +7,15 @@ const MINI_SIZE = 310;
 let mini = false;
 let oldSize;
 
+const I3IpcHelper = remote.getGlobal('I3IpcHelper');
```

Comment:
Is there really no better way to do this than using `getGlobal`.   For some reason it seems a hacky workaround :cry: 

I'm still probably going to merge it but instead of using `getGlobal` is it possible to use the `Emitter` classes?  Or is that not going to work for some reason?